include/service/entities/article_service.js

```diff
@@ -191,6 +191,28 @@ ArticleService.prototype.find = function(where, options, cb) {
 };
 
 /**
+ * Updates articles
+ *
+ * @param {String} articleId	id of article
+ * @param {Object} options	
+ * @param {Object} fields	fields to update	
+ * @param {Function} cb      Callback function
+ */
+ArticleService.prototype.update = function(articleId, options, fields, cb) {
+	var where = {
+		_id: articleId
+	}
+	
+	var dao  = new pb.DAO();
+	dao.updateFields('article', where, fields, options, function(err, result) {
```

Comment:
The article service has the ability to operate on articles and pages.  The type can be set for each instance of the service.  Instead of hard coding ```'article'```` use ```this.getContentType()```.